vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -165,8 +165,25 @@ function Test-RegistryAdded {
     }
 }
 
+function Test-PackagesPlugins {
+    $expectedPlugins = @(
```

Comment:
Are these package installed in current VHD? We may need to consider to avoid installing these packages.

Reply:
Yes, the pipeline installs them.
1. AntimalwareSignature. If we need to remove it, I will investigate that how to avoid installing it.
2. RunCommandWindows. I think we can ignore it (just keep it in code), it exists in this pipeline process, and it is automatically removed from Windows VM & AKS Cluster Node.

Reply:
You need to update both PR title and commit message to start with "test:". `extensions` may be better than `Plugins`.